images/macos/provision/core/graalvm.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e -o pipefail
+source ~/utils/utils.sh
+
+echo Installing GraalVM...
+brew install --cask graalvm/tap/graalvm-ce-java11
+
+exportPath=$(echo /Library/Java/JavaVirtualMachines/graalvm-ce-java*/Contents/Home/bin)
+echo "Setting up GRAALVM_11_ROOT variable to ${exportPath}"
+echo "export GRAALVM_11_ROOT=${exportPath}" >> "${HOME}/.bashrc"
+
+# GraalVM not being signed and notarized yet on macOS Catalina
+if ! is_Catalina; then
```

Comment:
Why is not if the comment says about Catalina?

Reply:
Indeed. Typo.